artifactory/commands/utils/result.go

```diff
@@ -54,21 +67,50 @@ func UnmarshalDeployableArtifacts(filePath, rtUrl string) (*Result, error) {
 		for _, artifact := range module {
 			if artifact.DeploySucceeded {
 				succeeded++
-				artifactsArray = append(artifactsArray, artifact.CreateFileTransferDetails(rtUrl))
+				artifactsArray = append(artifactsArray, artifact.CreateFileTransferDetails(url, repo))
 			} else {
 				failed++
 			}
 		}
 	}
-	err = clientutils.SaveFileTransferDetailsInFile(filePath, &artifactsArray)
+	err = clientutils.SaveFileTransferDetailsInFile(deployableArtifactsFilePath, &artifactsArray)
 	// Return result
 	result := new(Result)
 	result.SetSuccessCount(succeeded)
 	result.SetFailCount(failed)
-	result.SetReader(content.NewContentReader(filePath, "files"))
+	result.SetReader(content.NewContentReader(deployableArtifactsFilePath, "files"))
 	return result, nil
 }
 
+func GetDeployerUrlAndRepo(modulesMap *map[string][]clientutils.DeployableArtifactDetails, configPath string) (string, string, error) {
+	repo := getTargetRepoFromMap(modulesMap)
+	vConfig, err := utils.ReadConfigFile(configPath, utils.YAML)
+	if err != nil {
+		return "", "", err
+	}
+	// Relevant deploy repository will be written by the buildinfo project in diplyableArtifacts file from gradle extractor v2.24.12.
+	// In case of a gradle project with a configuration of 'usePugin=true' its possible that an old buildinfo version is being used.
+	// Deploy repository will be read from the configuration file.
```

Comment:
Deploy repository will be read from the configuration file.
-->
In this case, the value of "repo" will be empty, and the deployment repository will be therefore read from the local project configuration file.